ops/storage-ddls/cassandra-setup.cql

```diff
@@ -76,90 +72,112 @@ CREATE TYPE features (
     sentiment frozen<computedsentiment>,
     gender frozen<computedgender>,
     entities frozen<set<computedentities>>
-);
+);  
 
 CREATE TABLE computedtiles (
+    tileid text,
     tilex int,
     tiley int,
     tilez int,
-    tileid text,
     periodstartdate timestamp,
     periodenddate timestamp,
     periodtype text,
     period text,
-    pipeline text,
-    sourceid text,
-    topic text,
-    topiclangcode text,
+    pipelinekey text,
+    externalsourceid text,
+    topiconjunctionid uuid,
     insertion_time timestamp,
     heatmap text,
     placeids frozen<set<text>>,
     computedfeatures frozen<features>,
-    PRIMARY KEY ((tilex, tiley, tilez), periodstartdate, periodenddate, pipeline, sourceid, topic)
+    PRIMARY KEY ((period, periodtype, pipelinekey, topiconjunctionid), externalsourceid, tilez, tilex, tiley, periodstartdate, periodenddate)
+);
+
+create table topiconjunctions(
+    topic text,
+    topiconjunctionid uuid,
+    conjunctivekeywordcombination map<text, int>,//map value is representative of the aggregated mention count displayed in the dashboard
+    PRIMARY KEY ((topic, topiconjunctionid))
 );
 
-CREATE INDEX ON computedtiles (tileid);
-CREATE INDEX ON computedtiles (period);
-CREATE INDEX ON computedtiles (computedfeatures);
-CREATE INDEX ON computedtiles (periodtype);
+CREATE INDEX ON topiconjunctions ( KEYS (conjunctivekeywordcombination) );
 
 CREATE TABLE computedplaces (
     placeid text,
-    placename text,
     periodstartdate timestamp,
     periodenddate timestamp,
     periodtype text,
     period text,
-    pipeline text,
-    sourceid text,
-    topic text,
-    topiclangcode text,
+    pipelinekey text,
+    externalsourceid text,
+    topiconjunctionid uuid,
     insertion_time timestamp,
     computedfeatures frozen<features>,
-    PRIMARY KEY ((placeid), periodstartdate, periodenddate, pipeline, sourceid, topic)
+    PRIMARY KEY ((period, periodtype, pipelinekey, topiconjunctionid, placeid), externalsourceid, periodstartdate, periodenddate)
 );
 
-CREATE INDEX ON computedplaces (period);
-CREATE INDEX ON computedplaces (computedfeatures);
-CREATE INDEX ON computedplaces (periodtype);
-
-CREATE TABLE computedtopics (
+CREATE TABLE populartopics (
```

Comment:
I may be misunderstanding, but in order to allow to track trends and their directions, can I suggest we also add a new table called `topictrends`? This way, we can track topic mention counts even if they are holding steady, and then find points in time when they have a noticeable rate change (and maybe make that independent from pipeline). What do you think?